drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -377,8 +377,10 @@ function DragAndDropTemplates(configuration) {
     };
 
     var sidebarTemplate = function(ctx) {
-        var showAnswerButton = null;
-        if (ctx.show_show_answer) {
+        var showAnswerButton = sidebarButtonTemplate(
+                "show-answer-button sr"
+            );
```

Comment:
The `sr` class generally used for screen reader support, so the text is visible to screen readers but hidden from view. I don't think that is the aim here. You should simply not render the button unless needed, instead of hiding it using classes.

Reply:
Class ```sr``` is already used with ```go_to_beginning_button``` in code to hide it. I re-used it to ensure code uniformity.

Reply:
Class `sr` is already used with `go_to_beginning_button` in code to hide it. I re-used it to ensure code uniformity.

Reply:
Class `sr` is already used with `go_to_beginning_button` in code to hide it. I re-used it to ensure code uniformity.

Reply:
Class `sr` is already used with `go_to_beginning_button` in code to hide it. I re-used it to ensure code uniformity.

Reply:
Class `sr` is already used with `go_to_beginning_button` in code to hide it. I re-used it to ensure code uniformity.

Reply:
Class `sr` is already used with `go_to_beginning_button` in code to hide it. I re-used it to ensure code uniformity.

Reply:
Class `sr` is already used with `go_to_beginning_button` in code to hide it. I re-used it to ensure code uniformity.

Reply:
Class `sr` is already used with `go_to_beginning_button` in code to hide it. I re-used it to ensure code uniformity.

Reply:
Class `sr` is already used with `go_to_beginning_button` in code to hide it. I re-used it to ensure code uniformity.

Reply:
Class `sr` is already used with `go_to_beginning_button` in code to hide it. I re-used it to ensure code uniformity.

Reply:
Class `sr` is already used with `go_to_beginning_button` in code to hide it. I re-used it to ensure code uniformity.